dynalab/handler/handler.py.template

```diff
@@ -41,8 +41,8 @@ class Handler(BaseDynaHandler):
             os.path.join(model_file_dir, "config")
         )
         self.model = MyModel(config)
-        self.model.load_state_dict(torch.load(model_pt_path, map_location="cpu"))
-        self.model.to(device)
+        self.model.load_state_dict(torch.load(model_pt_path, map_location=device_str))
+        self.model.to(torch.device(device_str))
```

Comment:
I think this is probably easier to read but this means a little change here in ongoing handlers @TristanThrush